run_tests.sh

```diff
@@ -0,0 +1,7 @@
st,RDFDatetimeTest,FilehashTest,XoredSearchingTest,GRRFuseTest,ClientCommsTest,FilterTests,AFF4SparseImageTest,ByteSizeTest,OSXEnumerateRunningServicesTest,TimelineTest,TestTimelines,TestPackedVersionedCollection,VFSDirectoryTest,FindTest,RekallTests,RDFIntegerTest,TestExecutePython,MethodTest,EmailPluginTest,PackedVersionedCollectionCompactorTest,OSXMemoryTest,HashFileStoreTest,ArtifactFlowTest,AFF4ObjectLabelsListTest,TestExports,AuthTest,ConfigLibTest,GrepTests,GrrKbTest,TestArtifactCollectors,TriggersTest,UsersTest,SshdConfigTest,TestLinuxMemory,DurationTest,ClientURNTests,SqliteDataStoreTest,DataStoreTest,AFF4ObjectLabelTest,TestCheckFlows,TDBDataStoreBenchmarks,ThreadPoolTest,AFF4GRRTest,ChromeHistoryTest,GetClientStatsActionTest,TestSearchFileContent,TestRegistryFinderFlow,AFF4LabelsIndexTest,ListProcessesTest,Ipv6UtilsTest,RDFStructsTest,BuildTests,TestPlistFlows,FileStoreHashTest,ApiConfigRendererTest,IEHistoryTest,RunningStatsTest,AdministrativeFlowTests,YamlConfigTest,RDFValueBaseTest,RDFBytesTest,FileExportPluginTest,TestCopyPathToFile,SqliteDataStoreCSVBenchmarks,RDFValueBenchmark,TDBDataStoreTest,TestLinuxMemoryAnalysis,StoreTests,DataStoreBenchmarks,HintsTests,RekallResponseConverterTest,WindowsRegistryParserTest,ClientUtilsTest,GRRTempFileTestDirectory,EmptyActionTest,FakeDataStoreBenchmarks,AverageMicroBenchmarks,TestListVolumeShadowCopies,WindowsPersistenceMechanismsParserTest,HashDigestTest,WMIParserTest,CollectionExportPluginTest,StatsTests,MasterTest,ApiAff4RendererTest,SendEmailTests,TestClientInterrogate,TestExportHuntResultsFilesAsArchive,RDFURNTest,LinuxSoftwareParserTest,ArtifactParserTest,TestFilesystem,HuntTest,BatchConverterTest,TestArtifactCollectorsInteractions,OSXQuarantineTest,CollectorTest,CollectionFilesExportPluginTest,StandardHuntTest,UserTests,TestSearchFileContentWithFixture,TestCollections,ArtifactParserTests,DarwinPersistenceMechanismsParserTest,RDFValueCollectionApiObjectRendererTest,WebHistoryFlowTest,HTTPDataStoreTest,EmbeddedRDFValueTest,DataAgnosticExportConverterTest,GRRSeleniumTest,SqliteDataStoreBenchmarks,TestSystemRootSystemDriveFallbackFlow,OutputpluginsTest,ConfigFileTest,FlowTest,RegistryFlowTest,ExportTest,MatchMethodTests,ResultSetTest,FakeDataStoreTest,HintDefinitionTests,TestAdministrativeFlows,GRRTempFileTestFilename,FilestoreStatsCronFlowTest,ForemanTests,ReportsTest,GRRBaseTest,TestEndToEndTestFlow,DictTest,TestArtifactCollectorsRealArtifacts,KnowledgeBaseUserMergeTest,HandlerTests,NetstatTest,BasicContextTests,ArtifactHandlingTest,ObjcTest,TestWindowsMemory,WindowsActionTests,GrepTest,SystemCronFlowTest,UtilsTest,DataStoreCSVBenchmarks,CheckLoaderTests,TDBDataStoreCSVBenchmarks,ArtifactKBTest,MultiShardedQueueManagerTest,ConfigActionTest,SearchTest,RegistryVFSTests,OSXDriverTests,RekallTestSuite,GeneralFlowsTest,FlowStateTest,TestCryptoTypeInfos,AFF4Benchmark,GRRArtifactTest,GRRFuseDatastoreOnlyTest,TestOSXFileParsing,AFF4SymlinkTest,FlowTestsBaseclass,ConditionTest,RekallVADParserTest
```

Comment:
No. The purpose of travis is to let you know when tests are failing not to render a nice green button. If some tests pass and some fail then this is how it is - no point turning tests off just to get that nice green feeling :).

We dont have to require all new PRs to be travis clean if that is not the case - turning off tests doesnt give you any information anyway since it hides the fact they are failing anyway. 

The C++ client will likely require its own test program to run as part of the travis setup and so does not really depend on these tests passing anyway.